source/crud/tests/db/db-aggregate.yml

```diff
@@ -0,0 +1,58 @@
+data: []
+
+tests:
+    -
+        description: "Aggregate with $currentOp"
+        operation:
+            name: db-aggregate
+            database_name: admin
+            arguments:
+                pipeline:
+                  - $currentOp: {allUsers: true, idleConnections: true}
+                  - $match:
+                      command.aggregate: {$eq: 1}
+                  - $project: {command: 1}
+                  - $project: {command.lsid: false}
```

Comment:
Are two `$project` stages necessary so that `{command: 1}` can implicitly exclude everything _except_ `_id` and `command`, and then this stage excludes the `command` document's `lsid` field alone?

Small nit, but I think it'd be consistent to use either integers or booleans for all [projection syntax](https://docs.mongodb.com/manual/reference/operator/aggregation/project/index.html). I'd suggest `{command.lsid: 0}` here since we're already using `1` above (and it's more concise).

Reply:
Yes, the two `$project` stages are necessary for the reason you identified.